deployment/maintain/rebase-helper.md

```diff
@@ -52,34 +55,34 @@ Rebase-helper provides an API which you can use either directly from Python, or
 Example of patching new sources and starting scratch builds with fedpkg.
  This returns task_ids. The bash equivalents are included for comparison:
  
-* ``Python API``
+* **Python API**
 
-```
-from rebasehelper.application import Application
-cli = CLI([‘--non-interactive, ‘--builds-nowait’, ‘-buildtool’, ‘fedpkg’, upstream_version])
-rh = Application(cli)
-rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
-rh.run()
-```
+   ```python3
+   from rebasehelper.application import Application
+   cli = CLI(['--non-interactive', '--builds-nowait', '--buildtool', 'fedpkg', 'upstream_version'])
+   rh = Application(cli)
+   rh.set_upstream_monitoring() # Switch rebase-helper to upstream release monitoring mode.
+   rh.run()
+   ```
```

Comment:
This has unnecessary indent, right?

Reply:
Oh, not at all, IMHO. "**Python API**" and "**BASH**" are two items in a bulleted list, the indent places the examples at the same level (inside their corresponding bullet points).

Reply:
Well, even in case that is reflected on the resulting page properly, I'd prefer using (sub-)sub-headlines instead of those bullet points.

Reply:
Mmm, actually in moving that part to a separate page I ended up raising everything a heading level, so now they will just be sub-sub. (Previously, the titles of the sectinons they occupy were already sub-sub. Now they're just subheads.) ...Done.